src/components/marker/EntryMarker.jsx

```diff
@@ -0,0 +1,18 @@
+import { Marker, Popup } from 'react-leaflet';
+import useStore from '../../lib/useStore.js';
+
+export default function EntryMarker() {
+	const entries = useStore(state => state.entries);
+	console.log(entries);
+	return (
```

Comment:
Are you sure, you need the console.log here?

Reply:
No, thank you. I have overlooked it. It's cleared now